app/admin/prizes.rb

```diff
@@ -2,4 +2,33 @@
 
 ActiveAdmin.register Prize do
   permit_params :name, :percentage, :image, :description, :identifier, :prize_type
+
+  controller do
+    def create
+      total_prizes_percentages = Prize.pluck(:percentage).sum
+      if total_prizes_percentages > 100 || total_prizes_percentages + params[:prize][:percentage].to_i > 100
```

Comment:
I think this first validation `total_prizes_percentages > 100` is redundant since you're already ensuring that when saving (create and update) the total sum is not over 100, so it seems that this case will never happen, because if `total_prizes_percentages > 100` then this plus the new value will for sure be above 100, right? (same thing for the update)

Reply:
Yes, I'll remove it

Reply:
Done